bot/seasons/evergreen/issues.py

```diff
@@ -3,55 +3,60 @@
         self.bot = bot
 
-    @commands.command(aliases=("issues",))
-    @override_in_channel()
-    async def issue(
-        self, ctx: commands.Context, number: int, repository: str = "seasonalbot", user: str = "python-discord"
-    ) -> None:
-        """Command to retrieve issues from a GitHub repository."""
-        api_url = f"https://api.github.com/repos/{user}/{repository}/issues/{number}"
-        failed_status = {
-            404: f"Issue #{number} doesn't exist in the repository {user}/{repository}.",
-            403: f"Rate limit exceeded. Please wait a while before trying again!"
-        }
-
-        async with self.bot.http_session.get(api_url) as r:
-            json_data = await r.json()
-            response_code = r.status
-
-        if response_code in failed_status:
-            return await ctx.send(failed_status[response_code])
-
-        repo_url = f"https://github.com/{user}/{repository}"
-        issue_embed = discord.Embed(colour=Colours.bright_green)
-        issue_embed.add_field(name="Repository", value=f"[{user}/{repository}]({repo_url})", inline=False)
-        issue_embed.add_field(name="Issue Number", value=f"#{number}", inline=False)
-        issue_embed.add_field(name="Status", value=json_data["state"].title())
-        issue_embed.add_field(name="Link", value=json_data["html_url"], inline=False)
+    @commands.command(aliases=("pr",))
+    async def issues(self, ctx: commands.Context, number: int, repository: str = "seasonalbot",
+                     user: str = "python-discord") -> None:
+        """Command to get issues/pull request from GitHub."""
+        url = f"https://api.github.com/repos/{user}/{repository}/issues/{str(number)}"
+        mergeURL = f"https://api.github.com/repos/{user}/{repository}/pulls/{str(number)}/merge"
 
-        description = json_data["body"]
-        if len(description) > 1024:
-            placeholder = " [...]"
-            description = f"{description[:1024 - len(placeholder)]}{placeholder}"
-
-        issue_embed.add_field(name="Description", value=description, inline=False)
+        async with self.bot.http_session.get(url) as r:
+            json_data = await r.json()
 
-        await ctx.send(embed=issue_embed)
+        if r.status in respValue:
+            return await ctx.send(f"[{str(r.status)}] {respValue.get(r.status)}")
+
+        if "issues" in json_data.get("html_url"):
+            if json_data.get("state") == "open":
+                iconURL = icons.get("issue")
+            else:
+                iconURL = icons.get("issue_closed")
+        else:
+            async with self.bot.http_session.get(mergeURL) as m:
+                if json_data.get("state") == "open":
+                    iconURL = icons.get("pull-request")
+                elif m.status == 204:
+                    iconURL = icons.get("merge")
+                else:
+                    iconURL = icons.get("pull-request-closed")
+
+        resp = discord.Embed(colour=0x6CC644)
```

Comment:
Why a hardcoded color rather than the previous green from Seasonalbot's constants?

Reply:
This is so small I would like to come and fix it later once we have a stored emojis section.

Reply:
The stored emojis section is going to be fixed before this PR is merged.

Reply:
Do you think it will be done before hacktoberfest ends? Just curious.

Reply:
Presumably GH won't take a month to respond to a question, so most likely yes.

PRs don't have to be merged to count for the event.

Reply:
```py
issue_open = "<:IssueOpen:629695470327037963>"
issue_closed = "<:IssueClosed:629695470570307614>"
pr_open = "<:PROpen:629695470175780875>"
pr_closed = "<:PRClosed:629695470519713818>"
pr_merged = "<:PRMerged:629695470570176522>"
```

Reply:
Changed.